### DISCUSSION TITLE
New Validators and Delegation

### DISCUSSION THREAD

**Discussion by enekoastorkiza** [kcc-community/any-advice-issue#3] [Q&A]
Hi, is known that for a new validator at least 32 staked KCS will be needed.

* As soon as we mount the node and it has all the blocks synced can we start staking Tesnet KCS into it ?
* How do we stake testnet KCS into the desired validator ?
* Which are or will be node rewards is it get to be a real validator (TOP29) ?

**Comment by viaweb3** [comment]
copy, will reply later

**Reply by viaweb3** [reply]
As soon as we mount the node and it has all the blocks synced can we start staking Tesnet KCS into it?

yes, you can stake your tokens to any existed validator.
or you can set up your own validator, but this needs to be voted.

**Reply by viaweb3** [reply]
Which are or will be node rewards is it get to be a real validator (TOP29) ?

KCS would not be minted when mine blocks, so the block reward only comes from the transaction fees in this block, which means there is no reward if it is an empty block. I think that the block reward can not cover the server hardware costing at the beginning.

**Reply by viaweb3** [reply]
How do we stake testnet KCS into the desired validator ?

There is no UI or web pages for now, and I will write some code snippets to show this process.
